sdk/node/doc/assets/css/main.css

```diff
@@ -1,871 +0,0 @@
-/*! normalize.css v1.1.3 | MIT License | git.io/normalize */
-/* ========================================================================== HTML5 display definitions ========================================================================== */
```

Comment:
are you referring to all the generated sdk docs? I thought that @smithbk removed all of that with his PR.